database/wireless_regdb.ksy

```diff
@@ -0,0 +1,196 @@
+    -orig-id: countries_count
+    type: u4
+  - id: authentication_size
+    -orig-id: signature_len
+    type: u4
+    doc: length (in bytes) of the signature at the end of the file
+instances:
+  realms:
+    pos: realms_ptr
+    type: realm
+    repeat: expr
+    repeat-expr: realms_count
+  authentication:
+    pos: _io.size - authentication_size
+    size: authentication_size
+    doc: digital signature
+types:
+  freq_range:
+    doc: |
+      everything is in in kHz
+      Values of zero mean "not applicable", i.e. the regulatory does not limit a certain value.
+    seq:
+     - id: start
+       type: u4
+     - id: end
+       type: u4
+     - id: max_bandwidth
+       type: u4
+  power_rule:
+    seq:
+      - id: max_antenna_gain
+        type: u4
+        doc: in mBi (100 * dBi)
+      - id: max_eirp
+        type: u4
+        doc: in mBi (100 * dBi)
+  rule:
+    seq:
+      - id: freq_range_ptr
+        type: u4
+      - id: power_rule_ptr
+        type: u4
+      - id: flags
+        type: flags
+    instances:
+      power_rule:
+        pos: power_rule_ptr
+        type: power_rule
+      freq_range:
+        pos: freq_range_ptr
+        type: freq_range
+  rule_ptr:
+    seq:
+      - id: ptr
+        type: u4
+    instances:
+      rule:
+        pos: ptr
+        type: rule
+  flags:
+    seq:
+      - id: reserved1
+        type: b20
+      - id: auto_bw
+        -orig-id: RRF_AUTO_BW
+        type: b1
+        doc: Auto BW calculations
+      - id: reserved0
+        type: b2
+      - id: no_ibss
+        -orig-id: __RRF_NO_IBSS
+        type: b1
+        doc: old no-IBSS rule, maps to no-ir
+      - id: do_not_initiate_radiation
+        -orig-id: RRF_NO_IR
+        type: b1
+      - id: ptmp_only
+        -orig-id: RRF_PTMP_ONLY
+        type: b1
+        doc: this is only for Point To MultiPoint links
+      - id: ptp_only
+       -orig-id: RRF_PTP_ONLY
+        type: b1
+        doc: this is only for Point To Point links
+      - id: dfs_required
+        -orig-id: RRF_DFS
+        type: b1
+        doc: DFS support is required to be used
+      - id: no_outdoor
+        -orig-id: RRF_NO_OUTDOOR
+        type: b1
+        doc: outdoor operation not allowed
+      - id: no_indoor
+        -orig-id: RRF_NO_INDOOR
+        type: b1
+        doc: indoor operation not allowed
+      - id: no_cck
+        -orig-id: RRF_NO_CCK
+        type: b1
+        doc: CCK modulation not allowed
+      - id: no_ofdm
+        -orig-id: RRF_NO_OFDM
+        type: b1
+        doc: OFDM modulation not allowed
+
+  rules_collection:
+    seq:
+      - id: count
+        -orig-id: reg_rule_num
+        type: u4
+      - id: ptrs
+        type: rule_ptr
+        repeat: expr
+        repeat-expr: count
+
+  realm:
+    -orig-id: regdb_file_reg_country
+    seq:
+      - id: two_letter_code
+        -orig-id: alpha2
+        type: str
+        size: 2
+        -doc-ref: https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
```

Comment:
```suggestion
        doc-ref: https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
```

The key `doc-ref` shouldn't be dash-prefixed.

Reply:
:facepalm: Fixed! Thanks.